web-client/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@cogment/cogment-js-sdk": "^1.26.0",
+    "@cogment/cogment-js-sdk": "^1.26.1",
```

Comment:
IIRC because of this update there's a tsignore somewhere that should be removable